pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/mercury/GetMercuryEligibleSamplesRoute.java

```diff
@@ -38,7 +38,8 @@ protected Object processRequest(Request request, Response response, String userI
         }
         String realm = queryParams.get(RoutePath.REALM).value();
         String userIdRequest = UserUtil.getUserId(request);
-        if (!UserUtil.checkUserAccess(realm, userId, "kit_sequencing_order", userIdRequest)) {
+        if (!UserUtil.checkUserAccess(realm, userId, "kit_sequencing_order", userIdRequest) &&
+        !UserUtil.checkUserAccess(realm, userId, "view_seq_order_status", userIdRequest)) {
             log.warn("User doesn't have access " + userId);
```

Comment:
Just want to be sure you want an && here, not ||?

Reply:
No, either of those accesses can see this page, so you should not have both in order to get the error

Reply:
Could we express this in a positive way?
If the user has the role kit_sequencing_order or the user has view_seq_order_status, allow them to do 
    the action, 
else 
   deny access